src/styles/blocks/container.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Remove 'flex-wrap: wrap;' and replace it with 'grid-template-columns: repeat(4, 1fr);' and 'grid-gap: 16px;' to create a grid with 4 columns.